docs/advanced/installation.md

```diff
@@ -59,14 +59,20 @@ brew install pixi
 ### Windows Installer
 
 We provide an `msi` installer on [our GitHub releases page](https://github.com/prefix-dev/pixi/releases/latest).
-The installer will download Pixi and add it to the path.
+The installer will download Pixi and add it to the `PATH`.
 
 ### Winget
 
 ```
 winget install prefix-dev.pixi
 ```
 
+### Download From GitHub Releases
+
+Pixi is a single executable and can be run without any external dependencies.
+That means you can manually download the suitable archive for your architecture and operating system from our [GitHub releases](https://github.com/prefix-dev/pixi/releases), unpack it and then use it as is.
+If you want `pixi` itself or the executables installed via `pixi global` to be available in your `PATH`, you have to take care of that yourself.
```

Comment:
```suggestion
If you want `pixi` itself or the executables installed via `pixi global` to be available in your `PATH`, you have to add the global bin dir to the `PATH`, this normally is `$HOME/.pixi/bin`.
You can check the location pixi uses by running `pixi info` -> Global -> Bin dir.
```

Reply:
Done in https://github.com/prefix-dev/pixi/pull/3653/commits/c0cc8988800ce71797966823a000031d8952f176